opentracing/src/main/java/opentracing/Span.java

```diff
@@ -65,29 +63,7 @@
   Span setTag(String key, Number value);
 
   /**
-   * {@code message} is a format string and can refer to fields in the payload by path, like so:
-   *
-   * <pre>{@code
-   *
-   * span.info("first transaction is worth ${transactions[0].amount} ${transactions[0].currency}",
-   *     ImmutableMap.of(
-   *       "transactions", asList(
-   *         Transaction.builder().amount(10).currency("USD").build(),
-   *         Transaction.builder().amount(11).currency("USD").build(),
-   *       )
-   *     )
-   * );
-   * }</pre>
-   *
-   * @param message {@link Formatter format string} that can refer to fields in the args payload.
-   * @param args arbitrary payload
-   */
-  // See https://github.com/opentracing/opentracing.github.io/issues/30 about parameterization
-  Span info(String message, Object... args);
-
-  /** Same as {@link #info}, but for warnings. */
-  Span warning(String message, Object... args);
-
-  /** Same as {@link #info}, but for errors. */
-  Span error(String message, Object... args);
+   * @todo
+   **/
+  Span event(String message, /* @Nullable */ Object payload);
```

Comment:
I like `event` as it's short and sweet, 

Shall we use separate methods or accept nulls?

    Span event(String message);
    Span event(String message, Object payload);
    Span event(String message, ZonedDateTime timestamp);
    Span event(String message, Object payload, ZonedDateTime timestamp);
OR

    Span event(String message, 
                         /*@Nullable*/ Object payload, 
                         /*Nullable*/ ZonedDateTime timestamp);
I would normally be in favour of separate methods, but for a porcelain and premature api i think i'm in favour of having just one method.